### PR TITLE
1st commit, my extension is approved by Alex Olson because the last p…

### PR DESCRIPTION
**Pull request by kevinlutoronto**
…art's 50 epochs really took a long time to run

## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)

Completing this assignment.

## What did you learn from the changes you have made?

Training the models and updating the optimizers and adam.

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?

Completing this assignment.

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?

Greatly. The model training really took a long time to finish.

## How were these changes tested?

None.

## Checklist
- [x] I can confirm that my changes are working as intended

### CODE REVIEW

**Review by tianyi21 — APPROVED** [review]
Hi Kevin!

Thank you for submitting your DL assignments.

Your A1 is now complete.

Thanks,
Tianyi [LS]

File: 02_activities/assignments/lab_1.ipynb
Comment (tianyi21) [inline 02_activities/assignments/lab_1.ipynb:1]: - b: Please always reinitialize your model with `model = Sequential() ...` to reset all model parameters in addition to `model.compile()`. Or, you are continuing the training from the previous trained state with a different model setup. You may check this with the accuracy of the first epoch.

**Review by tianyi21 — APPROVED** [review]
Hi Kevin!

Thank you for submitting your DL assignments. Please refer to my comments attached to each file.

Your Lab 1-3 are now complete.

Thanks,
Tianyi [LS]
